### PR TITLE
This finishes the comparison calculator.

### PR DESCRIPTION
**Pull request by Tankenstein**
Adds a controller, adds a fundprovider to get estonian funds in the calculation, adds integration to epis.

### CODE REVIEW

File: src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@RestController
+@RequestMapping("/v1")
+@RequiredArgsConstructor
+public class FundComparisonController {
+
+    private static final Instant DEFAULT_TIME = parseInstant("1900-01-01");
+
+    private final FundComparisonCalculatorService fundComparisonCalculatorService;
+
+    @ApiOperation(value = "Compare the current user's return to estonian and world average")
```
Comment (oliverviljamaa) [inline src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java]: That makes it seem like it expects a return as an input. How about `Gets current user's, Estonian average, and world average return percentages`?

File: src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @GetMapping("/fund-comparison")
+    public FundComparison getComparison(
+            @ApiIgnore @AuthenticationPrincipal AuthenticatedPerson authenticatedPerson,
+            @RequestParam(value = "from", required = false) @DateTimeFormat(pattern="yyyy-MM-dd") Date startDate) {
```
Comment (oliverviljamaa) [inline src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java]: Is this a standard date time format for APIs?

File: src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java (hunk 1 above)
Comment (Tankenstein) [reply src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java]: Does not get world average

File: src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java (hunk 2 above)
Comment (Tankenstein) [reply src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java]: iso strings are but internally we have seemed to use this

**Pull request merged**